integrators/ordinary_differential_equations.hpp

```diff
@@ -132,11 +135,15 @@ class FixedStepSizeIntegrator : public Integrator<DifferentialEquation> {
   serialization::FixedStepSizeIntegrator::Kind const kind_;
 };
 
-enum class TerminationCondition {
-  Done,
-  ReachedMaximalStepCount,
-  VanishingStepSize,
-};
+// Any status not listed here is considered "severe" by the |Ephemeris|.
```

Comment:
Put that information in ephemeris itself; let's not refer to `physics` from its dependencies, that way madness lies.